tests/extensions/inline/italic/full/invalid/nested-deep.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="../extra-content.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="../extra-content.rng" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<?expected-error line="7" message="Italics must not be nested"?>
```

Comment:
seems like a good error message and easy to locate

Reply:
Of course it relies on the document not being on a single line

Reply:
You mean the line number?

Reply:
With Schematron we should be able to get the path to the Node. Can't with RNG as libxml doesn't (it can't even produce the column).